Add docs for first time setup (#6)
nerves-hub/documentation: nerves-hub/custom-deployment/run-migrations.md

# Run Migrations

The last thing to do is run initial migrations to configure the DB and seed some starter data.

### Default [VPC](https://aws.amazon.com/vpc/)

> A brand new AWS account will have a default VPC that has no name value. Our migration scrips will attempt to search VPCs by name and will throw an error if an unnamed VPC is encountered. Please give any unnamed VPCs some default name value.

Reviewer: ```suggestion
A brand new AWS account will have a default VPC that has no name value. Our migration scripts will attempt to search VPCs by name and will throw an error if an unnamed VPC is encountered. Please give any unnamed VPCs some default name value.
```

### Migrate

> It's time to run the migration scrips there are two: one in the NervesHub CA repo and one in the NervesHub WWW repo. In the CA repo run:

Reviewer: ```suggestion
There are 2 migration scripts which need to be run: one in the NervesHub CA repo and one in the NervesHub WWW repo. In the `nerves_hub_ca` repo directory run:
```

```bash
rel/scripts/ecs-migrate.sh nerves-hub-staging nerves-hub-staging nerves-hub-staging-ca nerves-hub-staging-ca-sg
```

> In the WWW repo run:

Reviewer: ```suggestion
In the `nerves_hub_www` repo directory run:
```

```bash
rel/scripts/ecs-migrate.sh nerves-hub-staging nerves-hub-staging nerves-hub-staging-www nerves-hub-staging-web-sg
```

These scripts may take a few minutes to run and will exit with a 0 status if they are successful.

### Current State

The app should now be totally accessible. The seeded login credentials will be:

```
username: nerveshub
password: nerveshub

> ```

Reviewer: We should prob add a statement here about removing this default seeded login once everything is setup.

Author: This is a good point which reminds me of something I realized isn't covered in these docs. Email is not totally working due to AWS sandboxing SMTP by default. I haven't gone through the process of figuring out what exactly needs to be done to enable emails e.g. forgotten password, etc. Do you think we should mention that in these docs for now or should we just update them once we know more?

Reviewer: 🤔 I think we've we know something will need extra work, its worth mentioning even if we're unsure how to actually fix. At least they will be aware to investigate themselves (and hopefully resolve and contribute back 🙌 )

Author: Sounds good. I'll add a note to the email section.
